### PR TITLE
Web interface documentation

### PR DESCRIPTION
**Pull request by bidoubiwa**
Documentation about the friendly web interface. Showcased with a gif and added to the features.

Fixes : meilisearch/MeiliSearch#407

### CODE REVIEW

File: advanced_guides/web_interface.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Web interface
+
+After adding documents to your MeiliSearch it is possible to try out the search engine with the integrated web interface.
+
+The **web interface is located on the same address and port as meilisearch**.
+
+
```
Comment (curquiza) [inline advanced_guides/web_interface.md]:
```suggestion
```

File: advanced_guides/web_interface.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Web interface
+
+After adding documents to your MeiliSearch it is possible to try out the search engine with the integrated web interface.
```
Comment (curquiza) [inline advanced_guides/web_interface.md]:
```suggestion
After adding documents to your MeiliSearch, it is possible to try out the search engine with the integrated web interface.
```

File: advanced_guides/web_interface.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+The **web interface is located on the same address and port as meilisearch**.
+
+
+## Example
+
+The server is listening on: `http://127.0.0.1:7700`.
+
+Lets add some movies.
```
Comment (curquiza) [inline advanced_guides/web_interface.md]:
```suggestion
Let's add some movies.
```

File: advanced_guides/web_interface.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+```bash
+curl -X POST  'http://127.0.0.1:8080/indexes/movies_uid/documents'\
+     --data @movies.json
+```
+
+Lets travel to `http://127.0.0.1:7700` in our browser.
```
Comment (curquiza) [inline advanced_guides/web_interface.md]:
```suggestion
Let's travel to `http://127.0.0.1:7700` in our browser.
```

Comment (curquiza) [reply advanced_guides/web_interface.md]: And "travel"? Maybe "go"?

File: getting_started/features.md [hunk 5]
```diff
@@ -20,3 +20,4 @@ Our feature list with extended explanations and links to the documentation.
 * **RESTfull API**
 * **Asynchronous write operations**: Operations will be added to a queue for **low response time** and **guaranteed consistency**.
 * [Bucket sort](/advanced_guides/bucket_sort.md) with [6 default ranking rules](/advanced_guides/ranking.md#ranking-rules) that [can be re-ordered](/advanced_guides/ranking.md#ranking-order) to suit your needs.
+* **Friendly web interface**: [Integrated web interface](/advanced_guides/web_interface) in MeiliSearch that let you try the search on the go.
```
Comment (curquiza) [inline getting_started/features.md]:
```suggestion
* **Friendly web interface**: [Integrated web interface](/advanced_guides/web_interface) in MeiliSearch that lets you try the search on the go.
```

File: advanced_guides/web_interface.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Web interface
+
+After adding documents to your MeiliSearch it is possible to try out the search engine with the integrated web interface.
+
+The **web interface is located on the same address and port as meilisearch**.
```
Comment (tpayet) [inline advanced_guides/web_interface.md]:
```suggestion
The web interface is served on **the same port** as MeiliSearch.
```

File: getting_started/features.md (hunk 5 above)
Comment (bidoubiwa) [reply getting_started/features.md]: I'm not convinced by your proposition but it made me realize I'm not convinced by my suggestion either. Either way, `lets` is grammatically correct and should be changed.

**Pull request merged**